src/Modules/GameModeUiModule/Services/GameModeUiModuleService.cs

```diff
@@ -2,148 +2,159 @@
 using EvoSC.Common.Services.Attributes;
 using EvoSC.Common.Services.Models;
 using EvoSC.Modules.Official.GameModeUiModule.Config;
+using EvoSC.Modules.Official.GameModeUiModule.Enums;
 using EvoSC.Modules.Official.GameModeUiModule.Interfaces;
+using EvoSC.Modules.Official.GameModeUiModule.Models;
 using Newtonsoft.Json;
 
 namespace EvoSC.Modules.Official.GameModeUiModule.Services;
 
-[Service(LifeStyle = ServiceLifeStyle.Transient)]
+[Service(LifeStyle = ServiceLifeStyle.Singleton)]
```

Comment:
Doesn't seem like it needs to be a singleton?

Reply:
Yeah it doesnt, I had made some changes that I reverted and forgot to change it back to transient.